server/controllers/allocationsController.ts

```diff
@@ -204,6 +204,26 @@ export default class AllocationsController {
     })
   }
 
+  async getDecisionEvidencing(_, res: Response, crn, staffTeamCode, staffCode, convictionNumber, pduCode) {
+    const response: OffenderManagerPotentialWorkload = await this.workloadService.getCaseAllocationImpact(
```

Comment:
It is a bit confusing that it uses the allocation impact method for decision evidence. 

If we don't have an endpoint that just returns the basic info then It might be worth making a new workloadService method that calls the same endpoint but has a more meaningful name.

Reply:
we've got https://hmpps-allocations-dev.hmpps.service.justice.gov.uk/webjars/swagger-ui/index.html#/unallocated-cases-controller/getUnallocatedCaseConfirmInstructions as well which would have the correct information too though I don't know what a more meaningful name would be

Reply:
Well in this case it's just the PoP and PP information we need so that endpoint's name `getUnallocatedCaseConfirmInstructions` is OK

Reply:
`/cases/unallocated/{crn}/convictions/{convictionNumber}/potential-allocation-details?staffCode={staffCode}`?